# settings
mode:           viz   # options: viz, sim, hdw
time_delta:     0.25

#                 Agent ID, Init position, Take-off height
# agent_init:     [['HB1', [-0.6, 0.9, 0.0], 0.4]]


# pick_loc:       [['P1', [0.4, 0.7, 0.4]]]

# drop_loc:       [['D1', [1.8, 0.6]]]

# agent_init:     [['HB1', [-0.6, 0.9, 0.0], 0.4],
#                  ['HB2', [-0.6, 0.3, 0.0], 0.4]]


# pick_loc:       [['P1', [0.4, 0.7, 0.4]],
#                  ['P2', [0.3, -0.2, 0.4]]]

# drop_loc:       [['D1', [1.8, 0.6]], 
#                  ['D2', [1.8, -0.3]]]

agent_init:     [['HB1', [-0.6, 0.9, 0.0], 0.4],
                 ['HB2', [-0.6, 0.3, 0.0], 0.4],
                 ['HB3', [-0.6, 0.3, 0.0], 0.4]]


pick_loc:       [['P1', [0.4, 0.7, 0.4]],
                 ['P2', [0.3, -0.2, 0.4]],
                 ['P3', [0.4, 0.7, 0.4]]]

drop_loc:       [['D1', [1.8, 0.6]], 
                 ['D2', [1.8, -0.3]],
                 ['D3', [1.8, 0.6]]]

map:
  dimensions:
    x_max: 21
    x_min: -9
    y_max: 11
    y_min: -9
  obstacles:
  - !!python/list
    - !!python/tuple
      - 0
      - -10
    - !!python/tuple
      - 12
      - -6
  - !!python/list
    - !!python/tuple
      - 0
      - 10
    - !!python/tuple
      - 12
      - 6